binancial/data/get_trades_realtime.py

```
@@ -1,45 +1,95 @@
+    # Apply nest_asyncio for async compatibility
+    nest_asyncio.apply()
+    
+    # Initialize socket id for proper cleanup
+    socket_id = None
+    
+    def handle_socket_message(msg: dict[str, Any]) -> None:
         try:
-            event_time = dt.datetime.fromtimestamp(msg['E'] / 1000)
-            event_time = event_time.strftime('%Y-%m-%d %H:%M:%S')
-            
+            event_time = datetime.fromtimestamp(msg['E'] / 1000)
```

Ruff (0.8.2)

`datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)